tests/typescript_comments/methods.ts

```diff
@@ -0,0 +1,34 @@
+export class Point {
+/**
+ * Does something.
+ */
+  foo() {}
+
+    /**
+     * Does something else.
+     */
+  bar() {}
+
+                /**
+                 * Does
+                 * something
+                 * much
+                 * better
+                 * than
+                 * the
+                 * rest.
+                 */
+  baz() {}
+
+      /**
+       * Buzz-Fizz.
+       * Note: This is indented too far.
+       */
+      fizzBuzz() {}
+
+      /**
+       * Turns the given string into pig-latin.
+       */
+              pigLatinize(value: string) {}
```

Comment:
No, if you put the comment inside of `pigLatinize()` then it's what I'm talking about.